libs/rollup.config.js

```diff
@@ -0,0 +1,55 @@
+import pkg from "./package.json";
```

Comment:
Excellent! Let's make sure that we make these things consistent with https://github.com/AudiusProject/fetch-nft/blob/main/rollup.config.ts